src/components/DragAndDropContext.tsx

```diff
@@ -13,6 +14,11 @@ const options = {
       backend: HTML5Backend,
       transition: PointerTransition,
     },
+  ],
+}
+
+if (isTouch && !isSafari()) {
+  options.backends.push(
```

Comment:
Please use inline `...isTouch && !isSafari ? [...] : []` syntax to avoid mutations.